### ISSUE TITLE
the loss decline too slow

### ISSUE THREAD

**Issue by DOVEXQ** [Tramac/Fast-SCNN-pytorch#51]
Hi, I use this project to train Cityscapes, use adam optim，lr is 3e-3,but the loss decline is too slow ,how can i solve this problem'
Thanks

**Comment by Alohomora0130** [comment]
Have you solve this problem? I meet the same situation.
